docs/synchronizations-faq.md

```diff
@@ -0,0 +1,43 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## What happens if you promote a synchronization to a primary synchronization?
+
+Private Packagist organizations that are synchronized with a remote organization have automatically all admins and owners of the synchronization assigned to the admins and owners team in Private Packagist.
+
+If an organization is synchronized with multiple remote organizations then only admins and owners of the primary synchronization are added to the respective teams.
+
+When promoting a synchronization to a primary synchronization, new admins and owners will be added from the primary synchronization to Private Packagist.
+In case you change the integration for the new primary synchronization (e.g. from GitHub to Bitbucket), users who are not in the new team will lose access to the organization. To restore their access, the user have to connect their account to the new service from the profile page.
```

Comment:
Can you reword this so it doesn't say `users who are not in the new team` but instead `users [or even admins and owners] who are not connected to the new service yet` lose admin/owner access? From what I have seen so far the problem usually was that the users Bitbucket (the new primary sync) account is listed as inactive in the admins/owners team. If the user is not shown in the admins/owners team anymore then this implies that they are not part of the team in Bitbucket and connecting the account won't help.